client/src/components/Feed/MessagesModal/MessagesContainer.js

```diff
@@ -230,51 +246,72 @@ export const PrivateMessageContainer = styled.div`
     position: relative;
     top: 0.28rem;
   }
-  span {
+  span.message {
+    font-family: Poppins;
     position: relative;
-    left: 0.7rem;
-    color: #939393;
+    left: 3rem;
+    bottom: 0.7rem;
+    font-size: 1.6rem;
+    color: white;
+    font-weight: 500;
+    line-height: 1.25rem;
     @media screen and (max-width: ${mq.phone.wide.maxWidth}) {
       display: none;
     }
   }
+  img.blueenvelope {
+    position: absolute;
+    top: 26%;
+    right: 70%;
+    width: 2.1rem;
+    display: block;
+    @media screen and (max-width: ${mq.phone.wide.maxWidth}) {
+      display: none;
+    }
+  }
+  img.whiteenvelope {
+    display: none;
+    @media screen and (max-width: ${mq.phone.wide.maxWidth}) {
+      position: absolute;
+      display:inline-block;
+      top: 22%;
+      right: 22%;
+      width: 3rem;
+      display: block;
+    }
+  }
+  
 
   img,
   span {
     pointer-events: none;
   }
-
-  ${(props) =>
-    props.isFromUserCard
-      ? `
-    margin-left: 7.4rem;
-  `
-      : ""}
-
   ${(props) =>
     props.isFromProfile
       ? `
-      display: flex;
-      align-items: center;
-      border: 0.1rem solid ${theme.colors.royalBlue};
+      position: absolute;
+      top: -3rem;
+      height: 6rem;
```

Comment:
I still think this is a little big. If I use the original figma numbers of 40px height and 152px width the button is smaller than how it appears in your design.  I used an online converted and did some tweaking and if you use root font-size 10 the conversion is more like 4rem for height, and 15.2rem for width

https://nekocalc.com/px-to-rem-converter